packages/core/src/components/callout/_callout.scss

```diff
@@ -87,6 +87,14 @@ $callout-padding-compact: $pt-grid-size;
     }
   }
 
+  &.#{$ns}-minimal,
+  &.#{$ns}-minimal.#{$ns}-intent-primary,
+  &.#{$ns}-minimal.#{$ns}-intent-success,
+  &.#{$ns}-minimal.#{$ns}-intent-warning,
+  &.#{$ns}-minimal.#{$ns}-intent-danger {
+    background-color: transparent;
+  }
+
   .#{$ns}-dark & {
     background-color: rgba($gray3, 0.2);
```

Comment:
It looks like we're missing overrides for the dark mode variants of Callout.

![Screenshot 2024-08-27 at 15 03 26](https://github.com/user-attachments/assets/cf974351-4507-40a3-8cf3-d615fdfd81d2)